src/requirements.txt

```diff
@@ -0,0 +1,33 @@
+django==3.2.13
+django-cors-headers==3.2.1
```

Comment:
I assume this matches the versions that were in the `requirements.txt` I edited in the root as part of the version upgrades refactoring?